corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -708,8 +721,8 @@ hqDefine("cloudcare/js/form_entry/form_ui", function () {
             });
```

Comment:
For the function above `childrenRequired`, does it still propagate correctly now that there could be additional levels of group nesting?

The purpose of this is really to find if any question contained in this `GroupedElementTileRow` is a required question

Reply:
https://github.com/dimagi/commcare-hq/pull/33834/commits/606fae2a2ee67bcdfef90adc3d5d9d126b334120